### PR TITLE
Add info about project purpose

### PR DESCRIPTION
**Pull request by j08lue**
- [x] Add a line about the purpose of this project
- [x] Add a table of supported services

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
 # Pyarc2stac
 
+A reverse proxy that dataset metadata from ESRI data services (ImageServer, MapServer, FeatureServer, SomethingServer) as [STAC](https://stacspec.org/),
```
Comment (smohiudd) [inline README.md]: The reverse proxy is actually [arc2stac](https://github.com/NASA-IMPACT/arc2stac). This repos is more for creating STAC collection using ArcGIS metadata. @amarouane-ABDELHAK can you confirm if this is correct?

Comment (amarouane-ABDELHAK) [reply README.md]: Yes [arc2stac](https://github.com/NASA-IMPACT/arc2stac) is the reverse proxy

File: README.md [hunk 2]
```diff
@@ -1,9 +1,19 @@
 # Pyarc2stac
 
+A reverse proxy that dataset metadata from ESRI data services (ImageServer, MapServer, FeatureServer, SomethingServer) as [STAC](https://stacspec.org/), 
+so they can become discoverable alongside other STAC-indexed resources.
```
Comment (j08lue) [inline README.md]:
```suggestion
Extract metadata from ESRI data services (ImageServer, MapServer, FeatureServer, SomethingServer) and convert it to [STAC](https://stacspec.org/), 
to make these services discoverable alongside other STAC-indexed resources.
```

File: README.md (hunk 1 above)
Comment (j08lue) [reply README.md]: Right! Accurate now?

**Review by smohiudd — APPROVED** [review]

**Pull request merged**